lib/android.js

```diff
@@ -668,9 +670,11 @@ exports.detect = function detect(config, opts, finished) {
 				var info = config['image.sysdir.1'] && systemImagesByPath[config['image.sysdir.1'].replace(/\/$/, '')];
 				if (info) {
 					var platform = platformsById[info.id];
-					target = platform.name + ' (API level ' + platform.apiLevel + ')';
-					sdk = platform.version;
-					apiLevel = platform.apiLevel;
+					if (platform) {
+						target = platform.name + ' (API level ' + platform.apiLevel + ')';
+						sdk = platform.version;
+						apiLevel = platform.apiLevel;
+					}
```

Comment:
Won't that result in a lot of "undefined" values in `ti info` like [here](https://github.com/appcelerator/node-titanium-sdk/pull/3/files#diff-b550f6a6b3ec7ae4e9b136a8f55e3954R686)?